modin/pandas/test/test_io.py

```diff
@@ -1641,6 +1642,27 @@ def test_to_parquet_keep_index(self, engine):
             engine=engine,
         )
 
+    def test_to_parquet_s3(self, s3_resource, engine, s3_storage_options):
+        # use utils_test_data because it spans multiple partitions
+        modin_path = "s3://modin-test/modin-dir/modin_df.parquet"
+        mdf, pdf = create_test_dfs(utils_test_data["int_data"])
+        pdf.to_parquet(
+            "s3://modin-test/pandas-dir/pandas_df.parquet",
+            engine=engine,
+            storage_options=s3_storage_options,
+        )
+        mdf.to_parquet(modin_path, engine=engine, storage_options=s3_storage_options)
+        df_equals(
+            pandas.read_parquet(
+                "s3://modin-test/pandas-dir/pandas_df.parquet",
+                storage_options=s3_storage_options,
+            ),
+            pd.read_parquet(modin_path, storage_options=s3_storage_options),
+        )
+        # check we're not creating local file:
+        # https://github.com/modin-project/modin/issues/5888
+        assert not os.path.isdir(modin_path)
```

Comment:
TBH this doesn't make sense, as such path won't _ever_ exist as a local path (because of its prefix); hence this check looks kinda useless to me

Reply:
I agree!